### ISSUE TITLE
Integrate OISD NSFW filter into your Adult API

### ISSUE THREAD

**Issue by BirdInFire** [AdguardTeam/AdGuardHome#6862]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Your adult website API Fail to block far too many website that are inside OISD NSFW list

### Proposed solution

Using the OISD NSFW list into your adult API

**Comment by Dynamic5912** [comment]
Just use the blocklist then. 

Sending data to AdGuard servers for checking parental controls and malware adds significant latency and reduces privacy.

**Comment by BirdInFire** [comment]
> If you have a list of sites that aren't being blocked, please send them to [support@adguard.com](mailto:support@adguard.com).

How about no ? in my simple test more than a dozen of them where missing ... they have a good reputation on removing very fast false positive, why would I manually send them domain when they can automate it ?

**Comment by BirdInFire** [comment]
> Just use the blocklist then.
> 
> Sending data to AdGuard servers for checking parental controls and malware adds significant latency and reduces privacy.

it's more to add more data to API for also other project who have a more robust api work (with preloaded list and private update)

**Comment by peytonjol** [comment]
I agree. I purchased adguard DNS for 1 year but i can't add custom and that list isn't available. It's too bad as it is a lot better for young people than only use Adguard Adult block.

Could you please add it ? : https://nsfw.oisd.nl Thanks.
